orchestration/dagster_orchestration/hca_orchestration/pipelines/load_hca.py

```diff
@@ -108,12 +108,27 @@ def import_start_notification(context: HookContext) -> None:
 
     context.resources.slack.send_message(blocks=key_value_slack_blocks("HCA Starting Import", key_values=kvs))
 
+@success_hook(
+    required_resource_keys={'slack', 'target_hca_dataset', 'dagit_config'}
+)
+def import_complete_notification(context: HookContext) -> None:
```

Comment:
🔧 I don't think this needs to be a hook but can rather go directly in the terminal solid code (or be called from there at least).

Reply:
Understood, this has been refactored to just send the slack message from the terminal solid